cpu/stm32/cpu_common.c

```diff
@@ -136,13 +147,19 @@ void periph_clk_dis(bus_t bus, uint32_t mask)
             RCC->APB1ENR1 &= ~(mask);
 #elif defined(CPU_FAM_STM32G0)
             RCC->APBENR1 &= ~(mask);
+#elif defined(CPU_FAM_STM32MP1)
+            RCC->MC_APB1ENCLRR |= mask;
```

Comment:
```suggestion
            RCC->MC_APB1ENCLRR &= ~mask;
```
?

Reply:
Thats the trick with the mp1, it uses clearing register RCC->MC_APB1ENCLRR:
https://www.st.com/resource/en/reference_manual/dm00327659-stm32mp157-advanced-armbased-32bit-mpus-stmicroelectronics.pdf#page=795

Reply:
Ah of course, nvm!

Reply:
Maybe add a short comment `/* write 1 to clear */` so this doesn't get 'fixed' accidentally in the future.